ci/ray_ci/automation/ray_wheels_lib.py

```diff
@@ -82,8 +92,9 @@ def download_ray_wheels_from_s3(
         directory_path: The directory to download the wheels to.
     """
     full_directory_path = os.path.join(bazel_workspace_dir, directory_path)
-
-    wheels = _get_wheel_names(ray_version)
+    _, minor_version, _ = ray_version.split(".")
```

Comment:
this might not always be 3 parts , maybe just take 2nd part without assuming it has 3 parts?

Reply:
oh ya I forgot about cases like `0.0.1.post1`. Updated to just grab the 2nd part